src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -531,6 +530,33 @@ protected void prepareSpawnArea(WorldServer world) {
     }
 
     @Override
+    public Optional<World> copyWorld(World originalWorld, String copyName) {
+        try {
+            FileUtils.copyDirectory(getWorldDir(originalWorld.getName()), getWorldDir(copyName));
```

Comment:
Might wanna hold of commenting on this till I push the more proper
implementation. This was a sample that worked well enough but wasn't really
thought about in a huge amount of detail.